n3fit/src/evolven3fit/evolve.py

```diff
@@ -22,6 +24,8 @@
     "level": logging.DEBUG,
 }
 
+NUM_CORES = multiprocessing.cpu_count()
```

Comment:
It should not be the default and it should *not* use every available core. At the very least it should be limited by the number of cores set in the runcard and controlled by a `--parallel` flag.

But, in any case, please let fix this in EKO.

Reply:
> It should not be the default and it should not use every available core. At the very least it should be limited by the number of cores set in the runcard and controlled by a --parallel flag.

For sure this can't be because the number of cores in the runcard are usually set as low as possible on the cluster (in order to run as many replicas as possible) while for the evolution one would want to run on as many cores as possible.

Anyway this was not really intended to be merged in the first place but I'd keep it around as it is handy.